From f6b25940d2e9b84bfd02de4bc0c7c584578a215a Mon Sep 17 00:00:00 2001
From: Andreas Mohr <and@gmx.li>
Date: Sat, 16 Jul 2016 17:31:08 +0000
Subject: [PATCH] (glob.c) Cleanup compile warning

glob.c:143:21: warning: implicit conversion loses integer precision: 'int' to 'char' [-Wconversion]
                c = ++cnt;
                  ~ ^~~~~
Signed-off-by: Andreas Mohr <and@gmx.li>
---
 lib/search/glob.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/search/glob.c b/lib/search/glob.c
index 905a671..b8040e1 100644
--- a/lib/search/glob.c
+++ b/lib/search/glob.c
@@ -116,7 +116,7 @@ static GString *
 mc_search__translate_replace_glob_to_regex (const char *str)
 {
     GString *buff;
-    int cnt = '0';
+    char cnt = '0';
     gboolean escaped_mode = FALSE;
 
     buff = g_string_sized_new (32);
